[2024-05-19T17:54:41.150+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T17:54:41.159+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: disney_api_to_s3.save_to_s3 scheduled__2024-05-19T16:35:00+00:00 [queued]>
[2024-05-19T17:54:41.162+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: disney_api_to_s3.save_to_s3 scheduled__2024-05-19T16:35:00+00:00 [queued]>
[2024-05-19T17:54:41.163+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2024-05-19T17:54:41.168+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_to_s3> on 2024-05-19 16:35:00+00:00
[2024-05-19T17:54:41.172+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=506) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T17:54:41.173+0000] {standard_task_runner.py:63} INFO - Started process 512 to run task
[2024-05-19T17:54:41.174+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'disney_api_to_s3', 'save_to_s3', 'scheduled__2024-05-19T16:35:00+00:00', '--job-id', '88', '--raw', '--subdir', 'DAGS_FOLDER/api_disney.py', '--cfg-path', '/tmp/tmp3yrxfswm']
[2024-05-19T17:54:41.175+0000] {standard_task_runner.py:91} INFO - Job 88: Subtask save_to_s3
[2024-05-19T17:54:41.197+0000] {task_command.py:426} INFO - Running <TaskInstance: disney_api_to_s3.save_to_s3 scheduled__2024-05-19T16:35:00+00:00 [running]> on host 620090466bc8
[2024-05-19T17:54:41.232+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='disney_api_to_s3' AIRFLOW_CTX_TASK_ID='save_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2024-05-19T16:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-19T16:35:00+00:00'
[2024-05-19T17:54:41.233+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T17:54:41.247+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-05-19T17:54:41.262+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T17:54:41.262+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T17:54:42.112+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T17:54:42.113+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T17:54:42.117+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=disney_api_to_s3, task_id=save_to_s3, run_id=scheduled__2024-05-19T16:35:00+00:00, execution_date=20240519T163500, start_date=20240519T175441, end_date=20240519T175442
[2024-05-19T17:54:42.146+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T17:54:42.154+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T17:54:42.155+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
